pkg/translate/service.go

```diff
@@ -122,20 +121,38 @@ func (t *TranslateServiceServer) CreateService(
 
 // ---------------------UpdateService-------------------------------
 
-var updateMaskAcceptablePaths = []string{"name"}
-
 type updateServiceParams struct {
 	mask    *fieldmaskpb.FieldMask
```

Comment:
Please, add `model.Mask` []string. Otherwise, the pb struct leaks into the core system.